src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/provisioner/main.py

```diff
@@ -34,25 +35,29 @@ def main():
             account_id = next(acc["Id"] for acc in all_accounts if acc["Name"] == account.full_name)
         except StopIteration: # If the account does not exist yet..
             account_id = None
-        create_or_update_account(organizations, account, adf_role_name, account_id)
+        create_or_update_account(organizations, support, account, adf_role_name, account_id)
 
 
-def create_or_update_account(org_session, account, adf_role_name, account_id=None):
+def create_or_update_account(org_session, support_session, account, adf_role_name, account_id=None):
     """Creates or updates a single AWS account.
     :param org_session: Instance of Organization class
     :param account: Instance of Account class
     """
     if not account_id:
         LOGGER.info(f'Creating new account {account.full_name}')
         account_id = org_session.create_account(account, adf_role_name)
+        # This only runs on account creation at the moment.
+        support_session.set_support_level_for_account(account, account_id)
+
     sts = STS()
     role = sts.assume_cross_account_role(
         'arn:aws:iam::{0}:role/{1}'.format(
             account_id,
             adf_role_name
-        ), 'delete_default_vpc'
+        ), 'adf_account_provisioning'
```

Comment:
Hey @sbkok. 
If you're referring to 'adf_account_provisioning' this isn't the name of the role. It will still use the adf-role (Be that the default organization access role / a custom one). 

This is only changing the role session name, to make it more generic, as this session is no longer used exclusively to delete the default vpc, and might even be used for more going forward. 

Hope this is okay, but otherwise I'm happy to revert this one line.

Reply:
Ah, my bad, I should have looked at those brackets more closely haha.
Thanks for your quick response and explanation, looks good.